src/components/layout/LayoutPageTitle.vue

```diff
@@ -35,10 +42,12 @@ export interface Props<T extends RoutesNamesList, P extends string> {
   isButtonEventCreateShown?: boolean
   title: string
   to?: NuxtRoute<T, P>
+  centered?: boolean
```

Comment:
Boolean properties should have the `is` prefix by convention.
Also, right now all properties in this codebase are sorted (or should be), so I'd go ahead and move this new property a few lines up.